docs/how-to.rst

```diff
@@ -0,0 +1,219 @@
+
+  $ pg_autoctl show state
+
+To analyze what's been happening to get to the current state, it is possible
+to review the past events generated by the pg_auto_failover monitor with the
+following command::
+
+  $ pg_autoctl show events
+
+.. hint::
+
+   The ``pg_autoctl show`` commands can be run from any node in your system.
+   Those command need to connect to the monitor and print the current state
+   or the current known list of events as per the monitor view of the system.
+
+   Use ``pg_autoctl show state --local`` to have a view of the local state
+   of a given node without connecting to the monitor Postgres instance.
+
+   The option ``--json`` is available in most ``pg_autoctl`` commands and
+   switches the output format from a human readable table form to a program
+   friendly JSON pretty-printed output.
+
+Inspecting and Editing Replication Settings
+-------------------------------------------
+
+When creating node it is possible to use the ``--candidate-priority`` and
+the ``--replication-quorum`` options to set the replication properties as
+required by your choice of Postgres architecture.
+
+To review the current replication settings of a formation, use the following
+command::
+
+  $ pg_autoctl get formation settings
+
+It is also possible to edit those replication settings at any time while
+your nodes are in production: you can change your mind or adjust to new
+elements without having to re-deploy everything. Just use the following
+commands to adjust the replication settings on the fly::
+
+  $ pg_autoctl set formation number-sync-standbys
+  $ pg_autoctl set node replication-quorum
+  $ pg_autoctl set node candidate-priority
+
+.. important::
+
+   The ``pg_autoctl get`` and ``pg_autoctl set`` commands always connect to
+   the monitor Postgres instance.
+
+   The ``pg_autoctl set`` command then changes the replication settings on
+   the node registration on the monitor. Then the monitor assigns the
+   APPLY_SETTINGS state to the current primary node in the system for it to
+   apply the new replication settings to its Postgres streaming replication
+   setup.
+
+   As a result, the ``pg_autoctl set`` commands requires a stable state in
+   the system to be allowed to proceed. Namely, the current primary node in
+   the system must have both its Current State and its Assigned State set to
+   primary, as per the ``pg_autoctl show state`` output.
+
+Implementing Maintenance Operations
+-----------------------------------
+
+When a Postgres node must be taken offline for a maintenance operation, such
+as e.g. a kernel security upgrade or a minor Postgres update, it is best to
+make it so that the pg_auto_failover monitor knows about it. This prevents a
+failover to target a node that's schedule to not be available anymore, and
+depending on your replication settings smoothly ensures that the primary
+node switches to async replication before Postgres is shut down on the
+secondary, avoiding write queries to be blocked.
```

Comment:
This sentence is very long and the last bit reads a bit weird (to me). I think it's god to split it and change it like:

This prevents ... anymore. Depending on your replication settings ~~smoothly~~**it also** 
ensures that the primary node switches

Reply:
Rephrased and split in two bullet points now for clarity.